spec/date.html

```diff
@@ -465,24 +468,25 @@ <h1>Properties of Temporal.Date Instances</h1>
       <li>[[ISOYear]], representing the year.</li>
       <li>[[ISOMonth]], representing the month within the year. It lies between 1 and 12, inclusive.</li>
       <li>[[ISODay]], representing the day within the month. It lies between 1 and DaysInMonth([[ISOYear]], [[ISOMonth]]), inclusive.</li>
+      <li>[[Calendar]], representing the calendar system to project the date into.</li>
     </ul>
-    <p>The value of each of these internal slots is a Number which is an integer.</p>
+    <p>The value of each of these internal slots is a Number which is an integer, except for [[Calendar]], whose value is a %Temporal.Calendar% object.</p>
   </emu-clause>
 
   <emu-clause id="sec-temporal-date-abstract-ops">
     <h1>Abstract Operations for Temporal.Date Objects</h1>
 
     <emu-clause id="sec-temporal-createtemporaldate" aoid="CreateTemporalDate">
-      <h1>CreateTemporalDate ( _isoYear_, _isoMonth_, _isoDay_ [ , _newTarget_ ] )</h1>
+      <h1>CreateTemporalDate ( _isoYear_, _isoMonth_, _isoDay_, _calendar_ [ , _object_ ] )</h1>
       <emu-alg>
         1. Perform ? RejectDate(_isoYear_, _isoMonth_, _isoDay_).
-        1. If ! DateTimeWithinLimits(_isoYear_, _isoMonth_, _isoDay_, 12, 0, 0, 0, 0, 0) is not ~in range~, then
-          1. Throw a *RangeError* exception.
-        1. If _newTarget_ is not given, set it to %Temporal.Date%.
-        1. Let _object_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%Temporal.Date.prototype%"`, « [[InitializedTemporalDate]], [[ISOYear]], [[ISOMonth]], [[ISODay]] »).
+        1. If ! DateTimeWithinLimits(_isoYear_, _isoMonth_, _isoDay_, 12, 0, 0, 0, 0, 0) is not ~in range~, throw a *RangeError* exception.
+        1. If Type(_calendar_) is not *Undefined*, throw a *RangeError* exception.
+        1. If _object_ is not given, set it to ? OrdinaryCreateFromConstructor(%Temporal.Date%, `"%Temporal.Date.prototype%"`, « [[InitializedTemporalDate]], [[ISOYear]], [[ISOMonth]], [[ISODay]], [[Calendar]] »).
```

Comment:
Let's keep the _newTarget_ argument, rather than duplicating the `OrdinaryCreateFromConstructor` call.